### ISSUE TITLE
Hydroxide reaction is not working

### ISSUE THREAD

**Issue by harrynull** [GTNewHorizons/GT-New-Horizons-Modpack#10262]
### Your GTNH Discord Username

harrynull#9630

### Your Pack Version

2.1.2.0

### Your Server

private server

### Type of Server

Don't know

### Your Expectation

Crafting a hydroxide cell with hydrogen and oxygen in a chemical reactor.

Expect a hydroxide cell to be crafted according to the recipe.
![image](https://user-images.githubusercontent.com/7413706/166510297-3198c907-419b-41da-9844-6cfee1316b73.png)

Note that this cell is required to craft prospector's scanner (MV).

### The Reality

No reaction is happening.

Supplying 2000L hydrogen, however, will results in distilled water, which has a very similar recipe. Is this caused by recipe conflict?
The distilled water recipe takes the same amount of power so it should rule out power issue or disabled machine.

### Your Proposal

Fix the recipe

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by TwoTailedFox** [comment]
This reaction is working for me in 2.1.2.3qf.

**Comment by chochem** [comment]
yeah also works fine for on 3qf. hydroxide with circuit on 3 and distilled water without.

Looks like this is fixed already.

**Comment by harrynull** [comment]
Thanks for confirming!
